python/rmmbr/serialization.py

```diff
@@ -0,0 +1,30 @@
+import json
+from hashlib import sha256
+from typing import Dict, List
+
+__all__ = ["Serializable", "FunctionSerializer"]
+
+
+Serializable = (
+    str
+    | int
+    | float
+    | bool
+    | None
+    | List["Serializable"]
+    | Dict["Serializable", "Serializable"]
+)
+
+
+class FunctionSerializer:
```

Comment:
you don't need a class here, because only `key_arguments` is used outside.

Reply:
👍